ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/launcher/AbstractPluginBlock.java

```diff
@@ -512,6 +512,17 @@ protected void createPluginViewer(Composite composite, int span, int indent) {
 			Object element = event.getElement();
 			if (element instanceof IPluginModelBase) {
 				handleCheckStateChanged(event);
+			} else if (element instanceof NamedElement) {
+				NamedElement namedElement = (NamedElement) element;
```

Comment:
optional suggestion to apply the Java 16 instance optimisation:
```suggestion
			} else if (element instanceof NamedElement namedElement) {
```